tests/integration/src/test/java/org/netxms/tests/SnmpTest.java

```diff
@@ -46,7 +46,10 @@ public void testSnmpObjectId() throws Exception
 		assertEquals(oid1, oid2);
 		
 		// Test toString
-		assertEquals(".1.3.6.1.2.1.1.1.0", oid1.toString());
+		assertEquals("1.3.6.1.2.1.1.1.0", oid1.toString());
```

Comment:
Pleae delete public void testSnmpObjectId() throws Exception test from file as I've added it to unit tests